src/neuroconv/tools/nwb_helpers/_configure_backend.py

```diff
@@ -53,13 +54,15 @@ def configure_backend(
                 dataset_name=dataset_name, data_io_class=data_io_class, data_io_kwargs=data_io_kwargs
             )
         # Special ndx-events v0.2.0 types
-        elif is_ndx_events_installed and isinstance(neurodata_object, ndx_events.Events):
-            neurodata_object.set_data_io(
-                dataset_name=dataset_name, data_io_class=data_io_class, data_io_kwargs=data_io_kwargs
-            )
+        elif is_ndx_events_installed and (get_package_version("ndx-events") == version.parse("0.2.0")):
+            if isinstance(neurodata_object, ndx_events.Events):
+                neurodata_object.set_data_io(
+                    dataset_name=dataset_name, data_io_class=data_io_class, data_io_kwargs=data_io_kwargs
+                )
         # But temporarily skipping LabeledEvents
-        elif is_ndx_events_installed and isinstance(neurodata_object, ndx_events.LabeledEvents):
-            continue
+        elif is_ndx_events_installed and (get_package_version("ndx-events") == version.parse("0.2.0")):
+            if isinstance(neurodata_object, ndx_events.LabeledEvents):
+                continue
```

Comment:
I think this can be merged into:

```python
# Special ndx-events v0.2.0 types
elif is_ndx_events_installed and (get_package_version("ndx-events") == version.parse("0.2.0")):
    # But temporarily skipping LabeledEvents
    if isinstance(neurodata_object, ndx_events.LabeledEvents):
       continue
    elif isinstance(neurodata_object, ndx_events.Events):
        neurodata_object.set_data_io(
          dataset_name=dataset_name, data_io_class=data_io_class, data_io_kwargs=data_io_kwargs
                 )
```